BankManagerConsoleApp/BankManagerConsoleApp/Bank/Bank.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Bank.swift
+//  BankManagerConsoleApp
+//
+//  Created by karen on 2023/07/14.
+//
+
+import Foundation
+
+struct Bank {
+    private var teller: Teller = Teller()
+    private var watingCustomer: Queue<Customer> = Queue<Customer>()
+    private var totalProcessingTime = ""
+    private var customerCount = UInt.zero
+    
+    mutating func open() {
+        lineUpCustomer()
+        totalProcessingTime = timeCheck {
+            dailyWork()
+        }
+        close()
+    }
+    
+    private mutating func lineUpCustomer() {
+        let customerCount = UInt.random(in: 10...30)
+        for number in 1...customerCount {
+            let customer = Customer(number: UInt(number))
+            watingCustomer.enqueue(customer)
+        }
+    }
```

Comment:
요거는 절대 잘못됐다거나 제가 suggestion에 쓴 방식을 권장한다는 아니구요,
그냥 요렇게도 쓸 수 있다 정도를 공유드리고 싶었던거라
그렇구나 정도만 보시면 될 거 같아요 ㅎㅎ
```suggestion
    private mutating func lineUpCustomer() {
        let customerCount = UInt.random(in: 10...30)
        let customers = (1...customerCount)
        customers
            .map(Customer.init)
            .forEach { watingCustomer.enqueue($0) }
            // mutating이 아니라면 
            // .forEach(watingCustomer.enqueue) 까지 줄일 수 있음
    }
```

Reply:
요게 절대 좋다는 말은 아니에요!
이게 더 가독성이 안좋을 수도 있어서
TPO에 맞게 잘 사용하는게 좋을 것 같아요 ㅎㅎ